docs/features/datastores/2_Using Datastores/2_creating-datastores.md

```diff
@@ -113,15 +113,14 @@ grid datastore create s3://ryft-public-sample-data/esRedditJson/ --name lightnin
 Example: 
 `grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy`
 
-In certain cases, your S3 bucket may fit one (or both) of the following criteria:
+Use `--no-copy` when you want to avoid creating dupicate copy of this data in your cluster account. Using this flag can significantly speed up Datastore creation time, It will prevent grid from making a copy of the dataset. 
```

Comment:
Was a typo and could be a little more concise see a possible example below.

"""Use `--no-copy` when you want to avoid creating a duplicate copy of the data in your cluster account. Using this flag can significantly speed up Datastore creation time by preventing the copy process."""

Reply:
Approved accidentally.